core/src/main/java/org/elasticsearch/indices/cache/query/terms/TermsLookup.java

```diff
@@ -56,21 +63,118 @@ public String getId() {
         return id;
     }
 
+    public String getPath() {
```

Comment:
it doesn't break anything given that this class was only used internally, it was not part of the java api. Only this change makes it part of the java api.